sdk/lib/crt/crt.cmake

```diff
@@ -441,6 +441,10 @@ if(ARCH STREQUAL "i386")
         list(APPEND CRT_ASM_SOURCE
             except/i386/cpp.s)
     endif()
+    if(USE_CLANG_CL)
+        list(APPEND CRT_ASM_SOURCE
+            stdlib/clang-alias.s)
+    endif()
```

Comment:
> as it can only be done by removing a compiler switch, which we have no CMake support for.

Isn't it https://stackoverflow.com/questions/28344564/cmake-remove-a-compile-flag-for-a-single-translation-unit that you want?

Reply:
That might work. Will have a look.

Reply:
Oh wow, this looks super complex. I feel like the current solution is good enough ;-)

Reply:
Hm? Seems it takes 3 lines to do it according to the example. Or am I missing something else?

Reply:
Well that is the "I tried this and it didn't work" part. The solution is like 50 lines of code.

Reply:
Ah ok; is it because we use an outdated CMake?